### PR TITLE
SEO-176496-Angular-page-with-redirect-links

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Please review the changes.

Thanks.

Regards,
Edith.

### CODE REVIEW

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angular/Gantt/Data-Binding.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ export class AppComponent {
 
     constructor() {
         this.dataManager = ej.DataManager({
-            url: "http://js.syncfusion.com/demos/ejServices/Wcf/TreeGridGantt/TreeGantt.svc/SelfReferenceDatas",
+            url: "https://js.syncfusion.com/demos/ejServices/Wcf/TreeGridGantt/TreeGantt.svc/SelfReferenceDatas",
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply angular/Gantt/Data-Binding.md]: resolved

File: angular/Gantt/Data-Binding.md [hunk 2]
```diff
@@ -229,7 +229,7 @@ export class AppComponent {
 
     constructor() {
         this.dataManager = ej.DataManager({
-            url: "http://js.syncfusion.com/demos/ejServices/Wcf/TreeGridGantt/TreeGantt.svc/SelfReferenceDatas"
+            url: "https://js.syncfusion.com/demos/ejServices/Wcf/TreeGridGantt/TreeGantt.svc/SelfReferenceDatas"
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply angular/Gantt/Data-Binding.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**